consensus/debugging.go

```diff
@@ -123,8 +123,8 @@ func (m *consensusModule) sendGetBlockStateSyncMessage(_ *messaging.DebugMessage
 
 	for _, val := range validators {
 		valAddress := cryptoPocket.AddressFromString(val.GetAddress())
-		if err := m.stateSync.SendStateSyncMessage(stateSyncGetBlockMessage, valAddress, requestHeight); err != nil {
-			m.logger.Debug().Msgf(typesCons.SendingStateSyncMessage(valAddress, requestHeight), err)
+		if err := m.stateSync.SendStateSyncMessage(stateSyncGetBlockMessage, typesCons.StateSyncGetBlockRequest, valAddress, requestHeight); err != nil {
+			m.logger.Error().Err(err).Msg("failed to send get block request state sync message")
```

Comment:
1. Option 1: s/get block request/`GetBlockRequest`
2. Option 2:
```go
m.logger.Error().Err(err)
    .Str("proto_type," "GetBlockRequest")
    .Msg("failed to send StateSyncMessage")
```

I kind of like option 2 but I'm not 100% sure if we should name this `message_type` or `proto_type` but might. Maybe worth asking in a shared channel?

Ditto elsewhere

Reply:
I agree that proto_type makes sense here, as message_type can be confusing with the HotStuffMessage, StateSyncMessage etc.proto